src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -123,6 +123,17 @@ public override void ExecuteCmdlet()
         [ValidateNotNullOrEmpty]
         public string AutomaticRepairGracePeriod { get; set; }
 
+        [Parameter(
+            Mandatory = false)]
+        public bool EnableAutomaticRepair { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Type of repair action (replace, restart, reimage) that will be used for repairing unhealthy virtual machines in the scale set. Default value is replace.",
+            ValueFromPipelineByPropertyName = true)]
+        [PSArgumentCompleter("Replace", "Restart", "Reimage")]
+        public string AutomaticRepairAction { get; set; }
```

Comment:
Please use ValidateSet attribute for input validation

Reply:
@vidai-msft , added the ValidateSet functionality.